src/MSONNamedTypeParser.h

```diff
@@ -15,6 +15,9 @@ using namespace scpl;
 
 namespace snowcrash {
 
+    /** MSON reserved characters matching regex */
+    const char* const MSONReservedCharsRegex = "[]:\()<>\{}[_*+`-]?";
```

Comment:
Should backtick really be in here? Since it's used to escape reserved characters. Wouldn't using a backtick would result in the following warning?

```
please escape the name of the data structure using backticks since it contains MSON reserved characters
```